### PR TITLE
Add continents and default country for the materials

### PR DESCRIPTION
**Pull request by magopian**
[user story](https://www.notion.so/Distances-Modification-de-l-affichage-du-pays-continent-et-du-calcul-de-distances-entre-filature-8a9d83a7ea1b4298a6eb60cca72f8305)

### CODE REVIEW

**Comment by magopian** [comment]
This should now be done and working as expected, what do you think @n1k0 on the review side and @paulboosz on the data/computation side?

Capture d’écran 2021-12-09 à 10 42 18

**Comment by magopian** [comment]
This needs some additional tests for materials with a different continent/country than Asie-Chine (because this PR passes the tests without modifying them, which is a sure sign of missing tests ;)

File: src/Page/Simulator.elm [hunk 1]
```diff
@@ -431,6 +431,7 @@ lifeCycleStepsView db displayMode simulator =
             (\index current ->
                 StepView.view
                     { db = db
+                    , inputs = simulator.inputs
                     , detailed = displayMode == DetailedMode
                     , index = index
                     , product = simulator.inputs.product
```
Comment (n1k0) [inline src/Page/Simulator.elm:437]: We don't need to expose `product` anymore as we now pass `inputs`.

**Review by n1k0 — COMMENTED** [review]
Looks great 👍 

A small nit to address and some tests and we're good to go. Great job!

**Comment by n1k0** [comment]
Also, be sure to merge/rebase with master before adding new tests, as the test suite structure as evolved substantially after #37.

**Comment by paulboosz** [comment]
I made a few tests with polyamide, polyethylene and check the transport. Looks good to me

**Comment by magopian** [comment]
@n1k0 I added some tests, let me know if you'd like some more ;)

File: src/Page/Simulator.elm (hunk 1 above)
Comment (n1k0) [reply src/Page/Simulator.elm]: Fixed in https://github.com/MTES-MCT/wikicarbone/pull/39/commits/1b42725f342ad46651f572ff63e3da664b20fe39

**Review by n1k0 — APPROVED** [review]
Looks good, great work 👍

**Pull request merged**